lib/ritm/proxy/launcher.rb

```diff
@@ -53,7 +53,8 @@ def build_proxy
       end
 
       def build_reverse_proxy
-        @https = Ritm::Proxy::SSLReverseProxy.new(@conf.ssl_reverse_proxy.bind_port,
+        @https = Ritm::Proxy::SSLReverseProxy.new(@conf.ssl_reverse_proxy.bind_address,
+                                                  @conf.ssl_reverse_proxy.bind_port,
```

Comment:
Allow access from the device in the same network, this is useful when you need to inception HTTPS between an iOS app and a remote server.